### ISSUE TITLE
Is this project an LTE pseudo base station?

### ISSUE THREAD

**Issue by TX107** [srsran/srsRAN#1070]
Is this project an LTE pseudo base station?

**Comment by tiger762** [comment]
YES! Very easy to set up and use. You'll need a SIM that you know the Ki and OPc as well as a handset ("UE"). I use a LG K7 with mine. You'll need at least an Ettus B200 SDR. Those sell for $1100 last time I checked although there are Chinese clones. What specific questions do you have?
